appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/internal/NetworkClient.java

```diff
@@ -138,11 +139,13 @@ public String generatePlayIntegrityChallenge(
     }
     URL url =
         new URL(String.format(PLAY_INTEGRITY_CHALLENGE_URL_TEMPLATE, projectId, appId, apiKey));
-    return makeNetworkRequest(url, requestBytes, retryManager);
+    // We don't forward the RetryManager, as we don't want a successful
```

Comment:
This is also going to stop invoking `retryManager.updateBackoffOnFailure()`, so now this generatePlayIntegrityChallenge call can spam our backend. Is that an acceptible tradeoff, or do we need to still pass the retryManager but pass in a flag instead? (which I know is more annoying, interface-wise).

Reply:
Good call, done.